source/dagger/gameplay/ping_pong/pingpong_playerinput.cpp

```diff
@@ -52,7 +52,7 @@ void PingPongPlayerInputSystem::Run()
         auto &t = view.get<Transform>(entity);
         auto &ctrl = view.get<ControllerMapping>(entity);
 
-        t.position.y += ctrl.input.y * s_PlayerSpeed * Engine::DeltaTime();
+        t.position.y += ctrl.input.y * s_PlayerSpeed * Engine::DeltaTime()*ConfigureSystem::PlayerSpeedMultiplier()*ConfigureSystem::GameSpeedMultiplier();
```

Comment:
I left you a comment in previous pull request. 
move this static fields to some non debug class and make imgui change them. 
Or add if defined(DAGGER_DEBUG) before using ConfigureSystem